### ISSUE TITLE
IMDb rating not working

### ISSUE THREAD

**Issue by gumida** [pkscout/metadata.tvshows.themoviedb.org.python#44]
It seems, like the IMDb rating parsing is no longer working. Instead of numbers my debug log returns just this:
```
2021-07-18 13:11:04.858 T:1998    DEBUG <general>: [metadata.tvshows.themoviedb.org.python (1.4.5)]: returning ratings of {}
2021-07-18 13:11:04.859 T:1998    DEBUG <general>: [metadata.tvshows.themoviedb.org.python (1.4.5)]: adding rating type of imdb
2021-07-18 13:11:04.859 T:1998    DEBUG <general>: [metadata.tvshows.themoviedb.org.python (1.4.5)]: adding rating of 0.0 and votes of 0
```
I assume, the regex in `imdbratings.py` is no longer working with the new website design?

**Comment by pkscout** [comment]
Yea, probably. This is why I hate scraping HTML for things. I'll take a look at the PR.

**Comment by pkscout** [comment]
Fixed with b194d0a1c88b67a9570fde564ea098e7fe5d5ce8. I will submit a PR for the scraper add-on repo today. It has to get reviewed and approved by someone else on the team, so it may take a few days depending on how busy folks are.
